changelog/unreleased/4435

```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix the Shares screen not updating.
```

Comment:
I would change the title: `Shares in non-root are updated correctly`
In the title we don't need the final point.